initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -88,24 +135,30 @@ elif [ -e /etc/redhat-release -o -e /etc/SuSE-release ]; then
 fi
 
 echo "Removing Directory - ${path}"
-rm -rf ${path}
+removedir ${path}
+
+if [ -n "${mklink}" -a -n "${lpath}" ] ; then
```

Comment:
It's used all the time by developers and testers - but should NOT be used by users - that's the only point I was making (in response to Michael's concern that this was a separate script users needed to know about...)